views/partials/svg/glyphs/passwords.hbs

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+<svg xmlns="http://www.w3.org/2000/svg" width="26" height="26" viewBox="0 0 26 26">
```

Comment:
Probably not relevant, but this is now the only SVG with a non 24x24 viewbox:

SVG | WIDTH | HEIGHT | VIEWBOX
:---|------:|-------:|-------:
bank-account-numbers.hbs | 24 | 24 | 0 0 24 24
credit-card-cvvs.hbs | 24 | 24 | 0 0 24 24
credit-cards.hbs | 24 | 24 | 0 0 24 24
dates-of-birth.hbs | 24 | 24 | 0 0 24 24
email-addresses.hbs | 24 | 24 | 0 0 24 24
geographic-locations.hbs | 24 | 24 | 0 0 24 24
historical-passwords.hbs | 24 | 24 | 0 0 24 24
ip-addresses.hbs | 24 | 24 | 0 0 24 24
more.hbs | 24 | 24 | 0 0 24 24
partial-credit-card-data.hbs | 24 | 24 | 0 0 24 24
_**passwords.hbs**_ | _**26**_ | _**26**_ | _**0 0 26 26**_
phone-numbers.hbs | 24 | 24 | 0 0 24 24
physical-addresses.hbs | 24 | 24 | 0 0 24 24
pins.hbs | 24 | 24 | 0 0 24 24
security-questions-and-answers.hbs | 24 | 24 | 0 0 24 24